front/src/gc-components/GCText.tsx

```diff
@@ -1,20 +1,20 @@
-import {ReactNode} from "react";
-import {DefaultMantineColor, MantineNumberSize, Sx, Text, TextProps} from "@mantine/core";
+import { ReactNode } from "react";
```

Comment:
Are just code styles changed here? You should use the same prettier settings as everybody in team. @Logovas1994 plz help here